native/result/result_test.go

```diff
@@ -0,0 +1,54 @@
+package result
+
+import (
+	"testing"
+
+	"github.com/goby-lang/goby/vm"
+)
+
+func TestEmptyResult(t *testing.T) {
+	r := vm.ExecAndReturn(t, `
+		require("result")
+		r = Result.empty()
+	`)
+
+	if r, ok := r.(*Result); ok {
+		if !r.empty {
```

Comment:
I don't think we should do verification in Go, we can add something like `#_empty?` as test helper. I actually prefer writing these cases using builtin test framework

Reply:
Maybe we can use a Go test function to start Goby spec then their coverage will be included?